### ISSUE TITLE
Make GitHub repo directly useable from Helm

### ISSUE THREAD

**Issue by hazcod** [netdata/helmchart#74]
Hi,
Until this is added to the official helm packages, it might be nice to have the helm Chart useable via this git repo as a helm repo?
I would like to install this using my terraform helm provider.
e.g. this should exist: https://raw.githubusercontent.com/netdata/helmchart/master/index.yaml

Ref https://github.com/netdata/helmchart/issues/72

**Comment by hazcod** [comment]
Another possibility is creating releases with `.tar.gz` archives so we can directly refer those.

**Comment by ncmans** [comment]
Definitely doable with something like Github pages. @cakrit what do you think? After Helm 3 release, [helm stable is not longer accepting new charts](https://github.com/helm/charts#status-of-the-project). Instead [Helm Hub](https://github.com/helm/hub/blob/master/Repositories.md) accepts inclusion of repositories which it then searches across.

**Comment by cakrit** [comment]
Don't ask for my permission gents, go ahead! :)

**Comment by prologic** [comment]
@hazcod Please put up a PR, I will happily review it!

**Comment by ncmans** [comment]
@prologic 
Here's my proposal on how we should do it (which applies to any helm chart for that matter):

1. Setup `gh-pages` as helm chart repo (ie create the branch and add some Readme) — this is the branch we will auto-update with new packaged releases of helm chart.
2. Build a CI pipeline that upon release tagging (ie `git tag v1.0.5`):
  a. Packages the helm chart (ie `helm package ...`)
  b. Creates a github release and uploads the packaged tarball as an asset.
  c. Updates the helm repo index on `gh-pages` branch (ie `helm repo index`) pointing to the correct github URL for the package.
  d. Pushes the `gh-pages` branch with the updated `index.yaml` (and maybe a changelog).

**Comment by prologic** [comment]
@ncmans Okay sounds good to me :) The `gh-pages` parts I already know quite well; so thanks for the `helm` hints. I guess @Ferroin and I should do this unless the OP is still interested :)

**Comment by hazcod** [comment]
Would love to, but i’m afraid i’m a bit too fresh to the Helm scene. 😄 appreciate the work!

**Comment by prologic** [comment]
@knatsakis I _think_ I've seen some PRs from you that are making this "in progress"? Added you as assigner and marked appropriately :)

**Comment by knatsakis** [comment]
@hazcod, @prologic a helm repo has been created at https://netdata.github.io/helmchart/

It has the bare minimum at the moment, and only includes the 3 latest versions of the helm chart. We have to manually update it for the time being. But it is usable from the terraform helm provider.

I will keep updating this issue with progress on this front.

**Comment by knatsakis** [comment]
Todo:
- Update documentation regarding the helm repo and in the helm repo itself.
- Automate updates to the helm repo as @ncmans described.

**Comment by prologic** [comment]
As a first step I would just create the CI workflow that implements what @MansM originally suggested and go from there. We can then "automate" this entirely once we figure out where it all fits in the greater schemes of things.

We can just manually trigger the workflow from `dispatc_event`(s) for now...

**Comment by fzyzcjy** [comment]
Hi any progress...?

**Comment by prologic** [comment]
None yet, hasn't been picked up from our Backlog. If you'd like to help I would be able to help you and get reviews done!

**Comment by fzyzcjy** [comment]
@prologic I will try to do that when having free time :)

**Comment by kam1kaze** [comment]
Hey folks. I suggest to update README.md file with HELM repo information, bcz I accidentally found it when I switched to gh-pages branch. It has existed for 6 months! :)

**Comment by OdysLam** [comment]
Thanks @kam1kaze for the suggestion. One of our engineers will respond promptly!
Take care :)

**Comment by prologic** [comment]
Hey @kam1kaze Its on my immediate backlog to get back to. I plan to complete this this week. Thanks for pining this!

**Comment by milindchawre** [comment]
+1
Would love to see it completed.

**Comment by OdysLam** [comment]
@milindchawre 

Thanks for the feedback! It's so important to us. 

P.S

Welcome to our community :)

**Comment by milindchawre** [comment]
Thanks @OdysLam 
Actually my plan was to [add netdata in civo marketplace](https://github.com/civo/kubernetes-marketplace/issues/89). By looking at its current installation steps, I decided to wait until this is fixed.

**Comment by prologic** [comment]
This is ready now and awaiting review #152 and #153

**Comment by milindchawre** [comment]
@prologic  Is it done? I don't see new installation steps in the [README.md](https://github.com/netdata/helmchart/blob/master/README.md#installing-the-chart) file.

**Comment by knatsakis** [comment]
Hi @milindchawre.

This is done, indeed. Kindly find the instructions here: https://netdata.github.io/helmchart/. #163 should also add a pointer to this in README.md.

**Comment by milindchawre** [comment]
Thanks @knatsakis for pointing it out.

**Comment by kam1kaze** [comment]
@knatsakis @prologic the instruction from https://netdata.github.io/helmchart/ doesn't work :)

```
$ helm repo add netdata https://helm.my-netdata.io/
$ helm install netdata
```

Domain helm.my-netdata.io could not be resolved:

```
❯ dig helm.my-netdata.io @8.8.8.8

; <<>> DiG 9.10.6 <<>> helm.my-netdata.io @8.8.8.8
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 59379
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;helm.my-netdata.io.            IN      A

;; AUTHORITY SECTION:
my-netdata.io.          1799    IN      SOA     barbara.ns.cloudflare.com. dns.cloudflare.com. 2034882277 10000 2400 604800 3600

;; Query time: 44 msec
;; SERVER: 8.8.8.8#53(8.8.8.8)
;; WHEN: Wed Oct 14 23:54:09 EEST 2020
;; MSG SIZE  rcvd: 112
```

**Comment by knatsakis** [comment]
Oh, sorry about that.

The correct command is:
```
helm repo add netdata https://netdata.github.io/helmchart/
```

(Fixed by #168)
